UIKit/Tasks/BirthdayReminderViewController.swift

```diff
@@ -0,0 +1,91 @@
+// BirthdayReminderViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Класс куда добавляются пользователи и информация о них
+final class BirthdayReminderViewController: UIViewController {
+    var people: [UserModel] = [
+        UserModel(name: "Helena Link", age: "25!", image: "woman", dayBirthday: "10.03", tillBirthday: ""),
+        UserModel(name: "Verona Tusk", age: "39", image: "womanOne", dayBirthday: "20.03", tillBirthday: "10 days"),
+        UserModel(name: "Alex Smith", age: "51", image: "man", dayBirthday: "21.04", tillBirthday: "42 days"),
+        UserModel(name: "Tom Johnson", age: "18", image: "boy", dayBirthday: "05.06", tillBirthday: "87 days"),
+    ] {
+        didSet { userAdding() }
+    }
+
+    // создание лейбла
+    private let birthdayReminderLabel: UILabel = {
+        let label = UILabel()
+        label.frame = CGRect(x: 78, y: 60, width: 209, height: 20)
+        label.text = "Birthday Reminder"
+        label.textAlignment = .center
+        label.font = UIFont.boldSystemFont(ofSize: 18)
+        label.textColor = .black
+        return label
+    }()
+
+    // создание изображения
+    private let cakeImageView: UIImageView = {
+        let image = UIImageView()
+        image.frame = CGRect(x: 314, y: 119, width: 44, height: 44)
+        image.image = UIImage(named: "cakeOne")
+        return image
+    }()
+
+    private var delegate: InformationDelegate?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        userAdding()
+        setupNavigationItem()
+    }
+
+    private func setupView() {
+        view.backgroundColor = .white
+        view.addSubview(cakeImageView)
+    }
+
+    private func setupNavigationItem() {
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            image: UIImage(systemName: "plus"),
+            style: .done,
+            target: self,
+            action: #selector(addButtun)
+        )
+    }
+
+    // метод для установки местоположения пользователя
+    private func userAdding() {
```

Comment:
методы должны отражать действие через глагол
```suggestion
    private func addUser() {
```